src/mavros_drone.py

```diff
@@ -0,0 +1,180 @@
+import roslibpy
+import numpy as np
+from drone import Drone
+from enum import Enum
+
+class MavrosDrone(Drone):
+
+    drone_type = "MavrosDrone"
```

Comment:
We should change either this drone_type value to "Mavros" or we should change the djimatrice_drone.py drone_type value to "DjiMatriceDrone" to stay consistent.